tests/legacy-cli/e2e/assets/protractor-saucelabs.conf.js

```diff
@@ -19,33 +19,24 @@ exports.config = {
   multiCapabilities: [
     {
       browserName: 'chrome',
-      version: '41',
+      version: 'latest',
       tunnelIdentifier,
     },
     {
-      browserName: 'chrome',
-      version: '75',
+      browserName: 'firefox',
+      version: 'latest',
       tunnelIdentifier,
     },
     {
       browserName: 'safari',
-      platform: 'OS X 10.11',
-      version: '9.0',
+      platform: 'OS X 10.13',
+      version: '12.4',
       tunnelIdentifier,
     },
     {
       browserName: 'safari',
-      platform: 'OS X 10.12',
-      version: '10.1',
```

Comment:
This was here to ensure that the nomodule polyfill works.

Reply:
I asked @IgorMinar if we should drop the polyfill in the first place since it’s an unsupported browser.

Reply:
Can we test the nomodules polyfill on ie?

Reply:
The polyfill is only used on Safari 10.1 (iOS Safari 10.3).  That specific version is in the unique situation of supporting module scripts but not the nomodule attribute.  This causes both sets of scripts (ES5 & ES2015) to be executed on page load.  Both the version prior (10.0) and the version after (11.0) do not have this issue.  Since 10.1 is technically out of the support range, the polyfill could be dropped and the polyfill script could be manually included by any application attempting to support 10.1.  (script link: https://gist.github.com/samthor/64b114e4a4f539915a95b91ffd340acc)

Reply:
sounds good to me. let's drop it.